### PR TITLE
feat(pci-public-ip): add manage additional ip service action link

### PR DESCRIPTION
**Pull request by seven-amid**
ref: DTCORE-2021

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | [DTCORE-2021](https://jira.ovhcloud.tools/browse/DTCORE-2021)
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by kqesar — APPROVED** [review]

File: packages/manager/apps/pci-public-ip/src/components/list/AdditionalIPServiceAction.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+
+export const AdditionalIPServiceAction = () => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+  const nav = useNavigation();
+
+  useEffect(() => {
+    nav
+      .getURL('dedicated', `#/ip?serviceName=_FAILOVER&page=1`, {})
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/src/components/list/AdditionalIPServiceAction.tsx:22]:
```suggestion
      .getURL('dedicated', `#/ip?serviceName=_FAILOVER`, {})
```
nitpick: page=1 probably not necessary

Comment (seven-amid) [reply packages/manager/apps/pci-public-ip/src/components/list/AdditionalIPServiceAction.tsx:22]: trying to be iso prod, i dont now the consequences of removing it

Comment (seven-amid) [reply packages/manager/apps/pci-public-ip/src/components/list/AdditionalIPServiceAction.tsx]: After test, there is no side effect if page param not assigned, i remove it finally

**Review by kqesar — CHANGES_REQUESTED** [review]
Fix conflict

**Pull request merged**